install.ps1

```diff
@@ -924,6 +924,14 @@ $backgroundImage = "${Env:VM_COMMON_DIR}\background.png"
 # Use background image for lock screen as well
 $lockScreenImage = "${Env:VM_COMMON_DIR}\lockscreen.png"
 Copy-Item $backgroundImage $lockScreenImage
+# Download FLARE VM (internet enabled) background image
+$backgroundInternetImage = "${Env:VM_COMMON_DIR}\background-internet.png"
+(New-Object net.webclient).DownloadFile('https://raw.githubusercontent.com/mandiant/flare-vm/main/Images/flarevm-background-internet.png', $backgroundInternetImage)
+# Download internet detection tool indicator icons
```

Comment:
I think the icons should be part of the tool or the tool package, as they shouldn't be configurable as the background image